src/http_server.cc

```diff
@@ -2737,7 +2737,12 @@ HTTPAPIServer::HandleInfer(
   }
 
   if (err != nullptr) {
-    LOG_VERBOSE(1) << "Infer failed: " << TRITONSERVER_ErrorMessage(err);
+    const char* request_id = "";
```

Comment:
Maybe the default can be "<id_unknown>"? If `TRITONSERVER_InferenceRequestId` ever returns error then we will print something instead of empty string

Reply:
Thanks! Updated.